docs/ota-client-guide/modules/ROOT/pages/meta-updater-usage.adoc

```diff
@@ -0,0 +1,87 @@
+= Usage
+:meta-updater-github-url: https://github.com/advancedtelematic/meta-updater/tree/master
+:metadata-expiry-article: xref:dev@ota-client::metadata-expiry.adoc[OTA Connect documentation]
+ifdef::env-github[:metadata-expiry-article: link:https://docs.ota.here.com/ota-client/dev/metadata-expiry.html[OTA Connect documentation]]
+
+//MC: No overlap with any content currently in the developer guide, but probably useful content to clean up and include. eg: use cases.
```

Comment:
Partially agree. I wonder if anyone has tested the commands in the OSTree section in years. That might not be worth keeping around anymore. And is the garage-push section really not already covered elsewhere? Doesn't seem too useful anymore. "aktualizr configuration" is useful but is largely already covered elsewhere, too. "aktualizr service resource control" might be unique. And the "garage-sign configuration" section was just what I wrote up before it made it to the docs portal; I think that's redundant now, too, right?

Reply:
Ditto above